.github/workflows/internal-build-fork.yml

```diff
@@ -0,0 +1,126 @@
+name: Run internal build (forks)
+
+# Handle fork PRs from Cloudflare org members
+# Uses pull_request_target to access secrets for fork handling
+on:
+  pull_request_target:
+
+concurrency:
+  # Cancel existing builds for the same PR.
+  # Otherwise, all other builds will be allowed to run through.
+  group: internal-build-fork.yml-${{ github.event.pull_request.number || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  # Handle forks from Cloudflare org members by creating mirror branches
+  handle-fork:
+    if: "${{ github.event.pull_request.head.repo.fork }}"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Cloudflare org membership
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          author="${{ github.event.pull_request.user.login }}"
+          echo "Fork detected. Checking if $author is a Cloudflare org member..."
+
+          if gh api orgs/cloudflare/members/$author --silent 2>/dev/null; then
+            echo "✓ Cloudflare org member confirmed"
+          else
+            echo "✗ Not a Cloudflare org public member"
+            echo ""
+            echo "This workflow only runs for forks from Cloudflare organization members."
+            echo "If you're an external contributor, please ask the auto-assigned reviewers"
```

Comment:
We no longer really have auto assigned reviewers. The Workers Runtime team will get tagged but not any individuals. I'm not quite sure how to handle that in the instructions though.